spec/forms/teacher_interface/further_information_request_item_text_form_spec.rb

```diff
@@ -14,7 +14,6 @@
   let(:response) { nil }
 
   it { is_expected.to validate_presence_of(:further_information_request_item) }
-  it { is_expected.to validate_presence_of(:response) }
```

Comment:
Why is this removed?

Reply:
If I don't remove it I get this error

![image](https://github.com/user-attachments/assets/74c767ae-483c-43ae-b410-3574f7456279)

and just searching around one ai chat bot said this

<img width="715" alt="Screenshot 2024-10-25 at 10 15 55" src="https://github.com/user-attachments/assets/0a2bcde2-f063-454e-9350-036ca172d233">

This might not be true because it's ai but I understood this to mean that this line I've removed is automatically checking and expecting the error message to be 'can't be blank' by default so I couldn't see a way of changing it. I didn't check any documentation for the 'shoulda-matchers' gem.